style.css

```diff
@@ -1,5 +1,258 @@
+  background-attachment: fixed;
+  background-size: cover;
+  background-position: center center;
+}
+
+.margin-t-l {
+  margin-top:100px;
+}
+
+
+.layout-container {
+  margin:20px 50px;
+}
+
+.home-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  font-family: "Oswald", sans-serif;
+  color: #202121;
+  /* margin: 0px 50px; */
+  height: 100vh;
+}
+
+.square {
+  background: rgba(1, 24, 40, 0.9);
+  border-radius: 5px;
+  font-weight: bold;
+  color: #f8f8f8;
+  font-size: 3rem;
+  text-align: center;
+  padding:0 30px;
+  
+}
+
+.button {
+  color: #f8f8f8;
+  border: solid 1px #f8f8f8;
+  border-radius: 5px;
+  text-decoration: none;
+  font-size: 1rem;
+  padding: 5px;
+  margin: 20px 100px;
+  display: flex;
+  justify-content: center;
+}
+
+.button:hover {
+  color: #f8f8f8;
+  border: solid 1px #f8f8f8;
+  border-radius: 5px;
+  text-decoration: none;
+  font-size: 1rem;
+  padding: 5px;
+  background: rgba(69, 175, 250, 0.748);
+  display: flex; /*CSS3*/
+  align-items: center; /*Vertical align*/
+  justify-content: center; /*horizontal align*/
+}
+
+.text {
+  font-weight: normal;
+  font-size: 1.5rem;
+}
+
+/* ABOUT */
+
+body {
+  margin: 0;
+  font-size: 16px;
+  font-family: "Oswald", sans-serif;
+  font-weight: 200;
+}
+
+.align-right{
   display: flex;
   justify-content: space-between;
-  padding: 20px 100px;
+  margin-left: auto;
+  width: 60%;
+  padding-left:20px;
+  box-sizing:border-box;
+}
+
+.nav-bar {
+  
+ width:100vw;
+ padding: 10px 50px;
+  border-bottom: solid #202121 1px;
+  background-color: white;
+  overflow: hidden;
+  position: fixed; /* Set the navbar to fixed position */
+  top: 0; /* Position the navbar at the top of the page */
+  box-sizing:border-box;
+}
+
+.nav-bar a {
+  text-decoration: none;
+ 
+  color: #202121;
+}
+
+.nav-bar a:hover {
+  font-weight: 400;
+  text-decoration: none;
+  color: #202121;
+}
+
+.container {
+  display: flex;
+  justify-content: space-around;
+  align-items: flex-start;
+  font-family: "Oswald", sans-serif;
+  color: #202121;
+  /* margin: 100px 50px; */
+}
+
+.img-profile {
+  max-width: 100%;
+  height: auto;
+}
+
+.main-left {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  text-align: center;
+  margin: 20px;
+  width: 40%;
+  min-height: 100%;
+}
+
+.main-right {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  text-align: justify;
+  margin: 20px;
+  width: 60%;
+  min-height: 100%;
+}
+
+footer {
+  background-color: #f8f8f8;
+  width: 100vw;
+  font-family: "Oswald", sans-serif;
+  text-align: center;
+  color: #202121;
+  padding: 10px 0px;
+  box-sizing:border-box;
+  left:0;
+}
+
+.table-tools {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-content: center;
+  align-items: center;
+}
+
+.img-icon{
+  width:30px;
+}
+
+.icon-column {
+  margin: 0px 2px 0px 20px;
+ 
+}
+
+.icon-block{
```

Comment:
Really nice work on making these columns!! Looks good. I would probably increase the width on smaller screens, so they stack underneath each other. Since you correctly applied flex-wrap:wrap property to the parent container they should cascade nicely when their size is increased (e.g. 100%).